portfolio/src/main/webapp/js/custom-script.js

```diff
@@ -716,3 +715,12 @@ function typewriterFeature() {
     if (MARTYR_NAMES.length) setTimeout(typeName, 2250);
   });
 }
+
+/* eslint-disable */
```

Comment:
Add comment indicating why linting is disabled here.

Reply:
Okay sure.